File: 2000 98 & ME/QuickStart/howto/samples/IO/anagrams/anagrams.cs
using System;
using System.IO;
using System.Collections;

class Anagrams 
{
    
    public static void Main(String[] args)
    {
        StreamReader din = File.OpenText ("words.txt");
        String str;
        StringTable st = new StringTable(); 
        
        Console.WriteLine("Reading data and insterting into a StringTable.");
        while ((str=din.ReadLine()) != null) {
            st.Add(str);
        }
        Console.WriteLine("Printing out the StringTable.");
        foreach (string s in st)
        {
            Console.WriteLine (s);
        }
        
		Console.WriteLine ("\r\nPress Return to exit.");
		Console.Read();
    }
}

File: 2000 98 & ME/QuickStart/howto/samples/IO/anagrams/vb/anagrams.vb
Imports System
Imports System.IO
Imports System.Collections

Module Anagrams

    
    Public Sub Main()
    
        Dim din As StreamReader = File.OpenText("words.txt")
        Dim s, str As String
        Dim st As StringTable = New StringTable
        
        Console.WriteLine("Reading data and insterting into a StringTable.")
        Do
            str = din.ReadLine()
            If str <> Nothing Then
                st.Add(str)
            End If
        Loop Until str = Nothing
        Console.WriteLine("Printing out the StringTable.")
        For Each s In st
            Console.WriteLine(s)
        Next
        Console.WriteLine()
        Console.WriteLine("Press Return to exit.")
        Console.Read()
    End Sub
End Module
